tools/exchange/src/main/resources/application.conf

```diff
@@ -58,6 +58,22 @@
       vertex: hive-field-0
       batch: 256
       partition: 32
+      # isImplicit: As a switch to allow implicit conversion of data, default true
+      # if isImplicit is true, the source field's datatype will be convert according to nebula data type.
+      #
+      # for Nebula String type
+      #     1. empty String will be convert to ""
+      #     2. Short, Int, Long, Decimal, Float, Double will be convert to String
+      #     3. Boolean true will be convert to String as format "true"
+      #     4. List/Array will be convert to String as format: [value1,value2]
+      #     5. Map will be convert to String as format: {key1:value1,key2:value2}
```

Comment:
may be is `{"key1":value1,"key2":value2}`

Reply:
it's "{key1:value1,key2:value2}"